tests/test_add_mds_to_cluster.py

```diff
@@ -42,8 +40,7 @@ def setup(self):
     Setting up the environment for the test
     """
     global CEPH_OBJ
-    CEPH_OBJ = cephfs.CephFileSystem(**self.fs_data)
-    CEPH_OBJ.create()
+    CEPH_OBJ = helpers.create_resource(**self.fs_data)
     assert POD.wait_for_resource(
```

Comment:
so now this needs to be removed as the create_resource is waiting.
Can you try to execute this test as a verifier?

Reply:
actually this is not really good to use create_resource as it needs more modification.
Missing selector so you won't be able to get only mds pods.
Just call OCS(kwargs) for now